docs-kits/kits/DCM-Kit/page_changelog.md

```diff
@@ -11,6 +11,22 @@ sidebar_position: 1
 
 All notable changes to this Kit will be documented in this file.
 
+## [0.2.0] - 2024-02-xx
+
+### Added
+
+- Documentation of DCM architecture (arc42)
+- Documentation versions based on Catena-X release
+
+### Changed
+
+- Documentation Adoption View based on Catena-X release
+- Documentation Development View based on Catena-X release
```

Comment:
We do not really understand what "based on Catena-X release" means - would be nice if you could become a bit more concrete here.

Reply:
Described in more detail